tests/puppeteer/pages/FO/home.js

```diff
@@ -29,63 +23,18 @@ module.exports = class Home extends CommonPage {
    * Check home page
    */
   async checkHomePage() {
-    await this.page.waitForSelector(this.logoHomePage, {visible: true});
-  }
-
-  /**
-   * go to the home page
-   */
-  async goToHomePage() {
-    await this.waitForSelectorAndClick(this.logoHomePage);
-    this.page.waitForNavigation({waitUntil: 'networkidle0'});
+    await this.page.waitForSelector(this.productImg.replace('%NUMBER', '1'), {visible: true});
```

Comment:
Maybe waiting for something that represent the home Page 
example : `body#index` or `section#content.page-home`